.github/workflows/publish.yml

```
@@ -39,15 +38,45 @@ jobs:
       - name: Run Postinstall Script
         if: steps.cache.outputs.cache-hit == 'true'
         run: npm run postinstall
+      - name: Check modified files
+        id: modified-files
+        uses: dorny/paths-filter@v2.9.2
+        with:
+          filters: |
+            core:
+              - 'libs/core/**'
       - name: Configure git
         run: |
           git config user.name "Kirby Bot"
           git config user.email "<>"
+      - name: Create npm config
+        if: steps.modified-files.outputs.core == 'true'
+        run:  |
+          cd libs/core 
+          echo '//registry.npmjs.org/:_authToken=${{secrets.NPM_PUBLISH_TOKEN}}' > ~/.npmrc
+      - name: Bump core package and update version in designsystem
```

```suggestion
      - name: Version bump core and update core dependency version in designsystem. 
```

Minor change but I think it makes

---

Right now it is done this way because we want to commit the changed version of the package bump together with the version update of designsystem. 

Do you think it is better to investigate how to save the version from the patch command in a global variable that we can access in a later step?  I had some problems doing this earlier with the approach used elsewhere in the other actions, but it should be doable.

---

Oh I remember we talked about that when looking into this! Just leave the step as is 😄